brands/amenity/fuel.json

```diff
@@ -2114,6 +2114,16 @@
       "name": "Susco"
     }
   },
+  "amenity/fuel|TA": {
```

Comment:
A full_name entry would be really useful here

Reply:
I wasn’t sure about this one. The company is called “TravelCenters of America”, but no one would ever refer to an individual location by that name. It’s always “TA Centerville” or somesuch. Perhaps `operator` and `operator:wikidata`?

Reply:
I always think of them as TA around here, but it would be nice for search to have the full name somewhere on the node that isn't displayed, but can be matched on